src/Bridge/Symfony/Bundle/Action/SwaggerUiAction.php

```diff
@@ -30,18 +31,20 @@
     private $resourceMetadataFactory;
     private $normalizer;
     private $twig;
+    private $oauth2Config;
     private $urlGenerator;
     private $title;
     private $description;
     private $version;
     private $formats = [];
 
-    public function __construct(ResourceNameCollectionFactoryInterface $resourceNameCollectionFactory, ResourceMetadataFactoryInterface $resourceMetadataFactory, NormalizerInterface $normalizer, \Twig_Environment $twig, UrlGeneratorInterface $urlGenerator, string $title = '', string $description = '', string $version = '', array $formats = [])
+    public function __construct(ResourceNameCollectionFactoryInterface $resourceNameCollectionFactory, ResourceMetadataFactoryInterface $resourceMetadataFactory, NormalizerInterface $normalizer, \Twig_Environment $twig, OAuth2Config $oauth2Config, UrlGeneratorInterface $urlGenerator, string $title = '', string $description = '', string $version = '', array $formats = [])
```

Comment:
New parameters must be appended at the end of the constructor and have a default value (`null` here) to preserve BC (users instantiate directly this class).